src/dataProvider/SparqlDataProvider.js

```diff
@@ -237,8 +237,8 @@ function generateContext(context) {
 function statusFetch(customFetch, context) {
   const wrappedFetchFunction = async (arg) => {
     try{
-      const response = await customFetch(arg);
-      context.fetchSuccess[arg] = true;
+      const response = await customFetch(arg); 
+      context.fetchSuccess[arg] = response.ok;  // <--- oplossing issue probleem hier, response is een object en heeft de prop "ok" die een bool is.
```

Comment:
Comments in English please. In this case, the comment is more or less redundant too. The code is very clear and the reader should understand it as such.